include/albatross/src/core/declarations.hpp

```diff
@@ -13,6 +13,12 @@
 #ifndef ALBATROSS_CORE_DECLARATIONS_H
 #define ALBATROSS_CORE_DECLARATIONS_H
 
+namespace mapbox {
+namespace util {
+template <typename... Ts> class variant;
```

Comment:
I'm not familiar with ellipsis in template names. What does this mean?

Reply:
It let's you provide an arbitrary number of template parameters.  Ie, `variant<int>` or `variant<int, double>` ...

Here's a good intro: https://eli.thegreenplace.net/2014/variadic-templates-in-c/